src/backend/src/utils/change-requests.utils.ts

```diff
@@ -165,3 +165,12 @@ export const calculateChangeRequestStatus = (
   }
   return ChangeRequestStatus.Open;
 };
+
+/**
+ * Determines whether all the change requests for a work package have been resolved or not
```

Comment:
Change this comment to be more accurate: Determines whether all the change requests in the array have been reviewed